src/translations/sqlb_nl.ts

```diff
@@ -0,0 +1,7344 @@
+        <translation>Initialisatiebestand (*.ini)</translation>
+    </message>
+    <message>
+        <location filename="../PreferencesDialog.cpp" line="711"/>
+        <source>The settings file has been saved in location :
+</source>
+        <translation>Het instellingenbestand is opgeslagen in:
+</translation>
+    </message>
+    <message>
+        <location filename="../PreferencesDialog.cpp" line="717"/>
+        <source>Open Settings File</source>
+        <translation>Instellingenbestand openen</translation>
+    </message>
+    <message>
+        <location filename="../PreferencesDialog.cpp" line="724"/>
+        <source>The settings file was loaded properly.</source>
+        <translation>Het instellingenbestand is correct geladen.</translation>
+    </message>
+    <message>
+        <location filename="../PreferencesDialog.cpp" line="731"/>
+        <source>The selected settings file is not a normal settings file.
+Please check again.</source>
+        <translation>Het gekozen instellingenbestand is geen normaal instellingenbestand. Controleer het nogmaals.</translation>
+    </message>
+</context>
+<context>
+    <name>ProxyDialog</name>
+    <message>
+        <location filename="../ProxyDialog.ui" line="14"/>
+        <source>Proxy Configuration</source>
+        <translation>Proxy-instellingen</translation>
+    </message>
+    <message>
+        <location filename="../ProxyDialog.ui" line="22"/>
+        <source>Pro&amp;xy Type</source>
+        <translation>Pro&amp;xytype</translation>
+    </message>
+    <message>
+        <location filename="../ProxyDialog.ui" line="35"/>
+        <source>Host Na&amp;me</source>
+        <translation>&amp;Hostnaam</translation>
+    </message>
+    <message>
+        <location filename="../ProxyDialog.ui" line="48"/>
+        <source>Port</source>
+        <translation>&amp;Poort</translation>
+    </message>
+    <message>
+        <location filename="../ProxyDialog.ui" line="68"/>
+        <source>Authentication Re&amp;quired</source>
+        <translation>&amp;Authenticatie vereist</translation>
+    </message>
+    <message>
+        <location filename="../ProxyDialog.ui" line="81"/>
+        <source>&amp;User Name</source>
+        <translation>&amp;Gebruikersnaam</translation>
+    </message>
+    <message>
+        <location filename="../ProxyDialog.ui" line="94"/>
+        <source>Password</source>
+        <translation>&amp;Wachtwoord</translation>
+    </message>
+    <message>
+        <location filename="../ProxyDialog.cpp" line="12"/>
+        <source>None</source>
+        <translation>Geen</translation>
+    </message>
+    <message>
+        <location filename="../ProxyDialog.cpp" line="13"/>
+        <source>System settings</source>
+        <translation>Systeeminstellingen</translation>
+    </message>
+    <message>
+        <location filename="../ProxyDialog.cpp" line="14"/>
+        <source>HTTP</source>
+        <translation></translation>
```

Comment:
I think it's a proper noun that doesn't need translation, so I think you've left the translation blank.
I also did like this in the past, but there was an comments that it is safer to copy the original text for proper nouns
that do not require translation. Please refer to [here](https://github.com/sqlitebrowser/sqlitebrowser/pull/2329#issuecomment-658202890) for details.